src/palette/view/widgets/specialcharactersdialog.cpp

```diff
@@ -662,7 +662,7 @@ void SpecialCharactersDialog::populateCommon()
         std::shared_ptr<FSymbol> fs = std::make_shared<FSymbol>(gpaletteScore->dummy());
         fs->setCode(id);
         fs->setFont(m_font);
-        m_pCommon->appendElement(fs, QChar(id));
+        m_pCommon->appendElement(fs, QString::fromUcs4(&id, 1));
```

Comment:
Mu3 just uses `QString(id)`

Reply:
I think that gives exactly the same problem, being equivalent to `QString(QChar(id))`. The `id` is namely implicitly converted to `QChar`, and then the `QString` constructor taking `QChar` is called. There is no `QString` constructor that takes anything like `char32_t` or `uint`. 

The reason that it was changed in master is that implicitly converting `char32_t` or `uint` to `QChar` is not allowed anymore (because of Qt 6, I think). The change in master simply made the conversion to `QChar` explicit, and the conversion from `QChar` to `QString` implicit, which is still allowed.

Reply:
I see, thanks

Reply:
OTOH: why not making those arrays `char16_t`? I don't see tham containing anything larger

Reply:
Ah, yes, the double sharp and double flat